nixos/modules/services/web-servers/quakejs.nix

```diff
@@ -0,0 +1,207 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkOption literalExpression;
+  inherit (lib.types)
+    lines
+    submodule
+    bool
+    uniq
+    str
+    port
+    ;
+  inherit (lib.lists) optionals;
```

Comment:
Please take the things from lib directly, so that we don't need to care about the internals of lib.